sample/sample-with-di/helloworldfeature/src/main/java/com/dropbox/kaiken/sample/advancedsample/helloworldfeature/HelloWorldFragmentAuthed.kt

```diff
@@ -0,0 +1,70 @@
+package com.dropbox.kaiken.sample.advancedsample.helloworldfeature
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import com.dropbox.kaiken.Injector
+import com.dropbox.kaiken.annotation.Injectable
+import com.dropbox.kaiken.runtime.InjectorFactory
+import com.dropbox.kaiken.runtime.InjectorHolder
+import com.dropbox.kaiken.sample_with_di.helloworldfeature.R
+import com.dropbox.kaiken.scoping.AuthAwareFragment
+import com.dropbox.kaiken.scoping.AuthRequiredActivityComponent
+import com.dropbox.kaiken.scoping.AuthRequiredActivityScope
+import com.dropbox.kaiken.scoping.DependencyProviderResolver
+import com.dropbox.kaiken.scoping.SingleIn
+import com.dropbox.kaiken.scoping.UserScope
+import com.squareup.anvil.annotations.ContributesTo
+import javax.inject.Inject
+
+fun DependencyProviderResolver.daggerInjectorAuth() =
```

Comment:
these 2 functions are all that are needed to wire the general auth activity scoped component

Reply:
Beautiful